@xen-orchestra/lite/src/components/pool/network/PoolNetworksPifStatus.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <UiInfo class="text-ellipsis" :accent="statusProps.accent">
```

Comment:
Based on my comment on `UiInfo`, you can add a class and change the icon alignment here:
```html
<template>
  <UiInfo class="pif-status">
</template>

<style lang="postcss" scoped>
.pif-status {
  align-items: center;
}
</style>
```